lib/libutee/include/pta_attestation.h

```diff
@@ -0,0 +1,22 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2018-2019, Linaro Limited
+ */
+#ifndef __PTA_ATTESTATION_H
+#define __PTA_ATTESTATION_H
+
+#include <util.h>
+
+/*
+ * Interface to the pseudo TA, which provides remote attestation.
+ */
+#define ATTESTATION_UUID \
+		{ 0xa2b0b139, 0x82dc, 0x4ffc, \
+			{ 0xa8, 0xa8, 0x7d, 0x7c, 0x63, 0x66, 0xe9, 0x84 } }
+
+#define TA_NAME		"attestation.pta"
```

Comment:
This isn't needed here.

Reply:
Fix [here](https://github.com/OP-TEE/optee_os/pull/4011/commits/e4e9394c7ddf7c87ef501988783c0906e46d7c7b).